gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.4-bin.zip
```

Comment:
I attempted to upgrade to gradle 8.8, but ran into breaking changes. If we want to upgrade to 8.8 we should tackle it in anothor PR.

Note using 7.6.4 works, but does prints the error `Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.`

Reply:
> If we want to upgrade to 8.8 we should tackle it in anothor PR.

Agreed!